lib/services/sorting/post/post_sort_option.dart

```diff
@@ -0,0 +1,28 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:proxima/models/database/post/post_firestore.dart";
+import "package:proxima/services/sorting/post/post_score_functions.dart";
+
+typedef PostScoreFunction = double Function(
```

Comment:
I think it is a bit misleading that `PostScoreFunction` is defined here and not in `post_score_function.dart`.
More generally speaking, what motivated you to separate `post_score_function.dart` in two files? I thought `hotScore(PostFirestore post, GeoPoint userPosition)` fitted well with the rest.

Reply:
Yes I agree with you the typedef of `PostScoreFunction` with the functions: done in 45e2bdf

I split the old `post_sort_option` in two file for three reasons:

1. Stay consistent with "same styles" enums in the project (like `MapSelectionOptions`, `NavigationbarRoutes`, ...)
2. Avoid polluting the global namespace when importing `PostSortOption` with global orphan functions. (instead accessed through the enum)
3. Separation of concerns: those functions could exist and be used without needing the enum so it makes sense to be able to reference them directly (as opposed to making them private for example).

Reply:
ok! By curiosity, how could `hotScore` be used outside of the post sorting?

Reply:
First of all, you are only talking about `hotScore`, but note that this change was not only for `hotScore`. It was made to regroup all 4 sorting functions together in `post_score_function.dart`: `hotScore`, `voteScore`, `dayScore`, `distanceScore`.

There are many use cases where those could be used without going through the more UI oriented associated enum `PostSortOption`.
To give a few examples, in some kind of data analysis scenario, or if we would like to get the score of posts to filter them, or apply some kind of spam detection, or maybe for the future heatmap, ...